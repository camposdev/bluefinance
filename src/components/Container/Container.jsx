import React from "react";
import PropTypes from "prop-types";
import styled from "styled-components";

const Wrapper = styled.div`
  width: 100%;
  margin: 0 auto;
  padding: 0 20px;

  @media (min-width: 768px) {
    width: 720px;
    padding: 0;
  }
  @media (min-width: 992px) {
    width: 960px;
  }
  @media (min-width: 1200px) {
    width: 1140px;
  }
`;

const Container = ({ children }) => <Wrapper>{children}</Wrapper>

Container.propTypes = {
  children: PropTypes.any.isRequired
}

export default Container;