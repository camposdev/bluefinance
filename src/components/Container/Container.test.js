import React from "react";
import renderer from "react-test-renderer";
import Container from "./Container";

describe("Container component", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Container><p>Lorem ipsum</p></Container>).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
