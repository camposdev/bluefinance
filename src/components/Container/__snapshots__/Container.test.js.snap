// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container component renders correctly 1`] = `
<div
  className="sc-bdVaJa GvrJR"
>
  <p>
    Lorem ipsum
  </p>
</div>
`;
